CMake/IntelDPCPPConfig-modified.cmake

```diff
@@ -207,7 +207,7 @@ if(SYCL_COMPILER)
   # Find Include path from binary
   find_path(SYCL_INCLUDE_DIR
     NAMES
-      CL/sycl.hpp
+      sycl.hpp
```

Comment:
Please ignore this when doing review. It has been taken care of by #5417. The diff will disappear when merging develop.